src/metabase/models/params/custom_values.clj

```diff
@@ -52,13 +52,16 @@
 
 (defn- values-from-card-query
   [card value-field query]
-  (let [value-base-type (:base_type (qp.util/field->field-info value-field (:result_metadata card)))]
+  (let [value-base-type (:base_type (qp.util/field->field-info value-field (:result_metadata card)))
+        expressions (get-in card [:dataset_query :query :expressions])]
     {:database (:database_id card)
      :type     :query
      :query    (merge
-                 {:source-table (format "card__%d" (:id card))
-                  :breakout     [value-field]
-                  :limit        *max-rows*}
+                 (cond-> {:source-table (format "card__%d" (:id card))
+                          :breakout     [value-field]
+                          :limit        *max-rows*}
+                   expressions
+                   (assoc :expressions expressions))
```

Comment:
Just curious: is the query map not allowed to have `{:expressions nil}` ? If it could, we wouldn't need the cond-> right?

I'm not bothered by doing it this way at all, though.

Reply:
Not sure, but this feels a little safer.